nvflare/tool/verify_startup_kits.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+from nvflare.lighter.constants import ProvFileName
+from nvflare.lighter.utils import verify_folder_signature
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-f", "--folder", type=str, help="folder to be verified")
+    parser.add_argument("-c", "--root_ca_path", type=str, help="root CA path")
+    args = parser.parse_args()
+    folder = args.folder
+    root_ca_path = args.root_ca_path
+    result = verify_folder_signature(
+        folder, root_ca_path, single_signer=True, signature_file=ProvFileName.SIGNATURE_JSON
+    )
+    if result is False:
```

Comment:
Should print result here for quick check. You can add an option to disable/enable the printing, .e.g -q (quiet) or -v (verbose).